From 586a80c87cff1bf8319f6b0f91a1be44a5e932a3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Tue, 1 Apr 2025 20:26:02 +0000
Subject: [PATCH] app: Don't free file_list

The list is already free'ed in the load_file_list() function. Freeing it
here also leads to a double free crash.
---
 src/gtr-application.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/gtr-application.c b/src/gtr-application.c
index 5e1e1554..8fe3c46c 100644
--- a/src/gtr-application.c
+++ b/src/gtr-application.c
@@ -597,7 +597,6 @@ gtr_application_setup_window (GApplication *application,
     {
       file_list = g_slist_reverse (file_list);
       gtr_actions_load_locations (window, file_list);
-      g_slist_free (file_list);
     }
 }
 
-- 
GitLab

